src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1152,6 +1443,28 @@ inline void rpp_load16_u8_to_f32_mirror_avx(Rpp8u *srcPtr, __m256 *p)
     p[1] = _mm256_cvtepi32_ps(_mm256_setr_m128i(_mm_shuffle_epi8(px, xmm_pxMask07To04), _mm_shuffle_epi8(px, xmm_pxMask03To00)));    /* Contains pixels 7-0 */
 }
 
+inline void rpp_store64_f32_to_u8_avx512(Rpp8u *dstPtr, __m512 *p)
+{
+    __m128i pxCvt[4];
+    __m512i px[4];
+    px[0] = _mm512_cvtps_epi32(p[0]);
+    px[1] = _mm512_cvtps_epi32(p[1]);
+    px[2] = _mm512_cvtps_epi32(p[2]);
+    px[3] = _mm512_cvtps_epi32(p[3]);
+    
+    pxCvt[0] = _mm512_cvtusepi32_epi8(px[0]);
+    pxCvt[1] = _mm512_cvtusepi32_epi8(px[1]);
+    pxCvt[2] = _mm512_cvtusepi32_epi8(px[2]);
+    pxCvt[3] = _mm512_cvtusepi32_epi8(px[3]);
+
+    __m512i out;
+    out = _mm512_inserti32x4(_mm512_setzero_epi32(), pxCvt[0], 0);
```

Comment:
why are we using _mm512_setzero_epi32() here? what is the use, please add comments

Reply:
Modified this to remove usage of insert function and used storeu functions instead